kb-search/KBSearch_Test.py

```diff
@@ -9,10 +9,10 @@
 
 os.environ[
     'KATECHEO_KB'] = 'faith=https://storage.googleapis.com/pachyderm-neuralbot/knowledge_bases/kb_gq_6k_general.json,health=https://storage.googleapis.com/pachyderm-neuralbot/knowledge_bases/kb_wedmd_health.json'
-os.environ['ARTICLE_ID'] = 'article_url'
-os.environ['ARTICLE_TITLE_KEY'] = 'title'
-os.environ['ARTICLE_BODY_KEY'] = 'body'
-os.environ['COSINE_SIMILARITY_THRESHOLD'] = '0.19'
```

Comment:
I wonder if we should decrease this threshold for the default, as I ended up using 0.15 in the paper. 0.19 filtered out a significant number of questions.

Reply:
My intention of keeping it at 0.19 was just that so our experiments would align with this value but since we can't publish the experiments because of the private corpus we are using, we can update it to any value for now...